templates/definition/tariff/pun.yaml

```diff
@@ -0,0 +1,20 @@
+template: pun
+products:
+  - brand: PUN
+params:
+  - name: charges
```

Comment:
Replace with

      - preset: tariff-base

and then

    {{ include "tariff-base" . }}